blue-hat/cbeer/blue_hat.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+  "flag"
+  "fmt"
+  "os"
+  "image"
+  "image/png"
+  "image/color"
+)
+
+func main() {
+  infilePtr := flag.String("infile", "red_hat.png", "Input file name")
+  outfilePtr := flag.String("outfile", "blue_hat.png", "Output file name")
+
+  flag.Parse()
+
+  fmt.Println("input file name:  ", *infilePtr)
+  fmt.Println("output file name: ", *outfilePtr)
+
+  infile, err := os.Open(*infilePtr)
+  if err != nil {
+    fmt.Fprintf(os.Stderr, "Failed to open input file: %v\n", err)
+    os.Exit(1)
```

Comment:
You can use `log.Fatal` here which logs the error and exits.

Reply:
I didn't know about that. Thanks!